src/deploy/NetworkConfiguration.ts

```diff
@@ -100,7 +101,7 @@ function getAssetConfigs(
     borrowCollateralFactor: percentage(assetConfig.borrowCF),
     liquidateCollateralFactor: percentage(assetConfig.liquidateCF),
     liquidationFactor: percentage(assetConfig.liquidationFactor),
-    supplyCap: number(assetConfig.supplyCap), // TODO: Decimals
+    supplyCap: BigInt(number(assetConfig.supplyCap)) * (10n ** BigInt(assetConfig.decimals)),
```

Comment:
Where do you see that? I don't believe they do (and are verified in the other PR)

Reply:
Example: https://github.com/compound-finance/comet/blob/main/deployments/kovan/usdc/configuration.json#L33

Reply:
I see what you mean, we should probably change these then? This seems problematic as JS numbers can't represent these numbers precisely, already I bet those supply caps are off (or would be, if we didn't truncate to a whole number of units in Comet)

Reply:
Yeah definitely, the lack of precision for larger numbers could be a problem. We use scaled numbers in for other values in the configuration too (e.g. `targetReserves`, `baseMinForRewards`). I don't think we've run into issues for the USDC deploy because those values are all less than `Number.MAX_SAFE_INTEGER`, but we won't be so lucky with the WETH deploy.

Reply:
Probably easiest if we just handle the conversion from scientific notation string to BigInt in this file. I don't think we can do the conversion natively in JS (`BigInt` constructor can take a string, but not scientific notation string), but we can write a custom converter that does it.

Reply:
I think we've already exceeded max safe on a number of these

Reply:
I added TODO comments for `targetReserves` and `baseMinForRewards` since they weren't necessary yet and trying to avoid unrelated changes, but I updated all supply caps in configuration.jsons